README.md

```diff
@@ -2,25 +2,48 @@
 
 [![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-data.svg?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-data)
 
+## How to create a project with Sunrise data
+
+Before starting the import, make sure you have access to the [Admin Center](https://admin.commercetools.com). You will also need to run the application in this repository, check that you meet the [requirements](#requirements) and [clone](https://help.github.com/articles/cloning-a-repository/) this repository to your computer.
+
+### 1. Create your project
+1. Open the [Admin Center](https://admin.commercetools.com) and create an empty project (without sample data).
+
+### 2. Prepare node environemnt for Import
+1. Go to the root of this project, were the package.json is located.
+2. Install all node dependencies
+    ```js
+        npm install
+    ```
+3. Set commercetools project credentials as npm config values:
+    <pre>
+        npm run set:authUrl <i>authUrl</i>
```

Comment:
We should add an example URL here.  
The Java tool is picky about the exact form - it fails hard without special info if there's a trailing slash  like `https://auth.commercetools.com/`    I don't know how the exact magic works but if possible just having to put the hostname here could help.